.gitignore

```diff
@@ -71,3 +71,10 @@ site/
 **/*.Rcheck/
 **/.Rhistory
 .Rproj.user
+/.vs
```

Comment:
I don't think you need to change the `.gitignore` file with this change. Can you explain why this is being proposed here?

If there is a C# specific pattern that needs to be ignored (like `.vs`), there is a `csharp/.gitignore` file for the C# specific patterns - but it already has `.vs`.

Reply:
if you click "Open in Visual Studio" on github - the root folder is opened and folders are created automatically... Any one else that does that will probably see those folders get created as well - so its just annoying to see those all the time..